kprogresshud/src/main/java/com/kaopiz/kprogresshud/KProgressHUD.java

```diff
@@ -159,8 +166,20 @@ public KProgressHUD setBackgroundColor(int color) {
         return this;
     }
 
+    /**
+     * Specify the HUD background color
+     *
+     * @param onKeyListener KeyListener
+     * @return Current HUD
+     */
+    public KProgressHUD setOnKeyListener(@Nullable DialogInterface.OnKeyListener onKeyListener) {
+        this.mProgressDialog.setOnKeyListener(onKeyListener);
```

Comment:
Just this is nice. You don't need to make any changes inside dialog class (I commented below).